test/test.js

```diff
@@ -50,6 +50,7 @@ describe('Basic functionality', function() {
       .get('/')
       .type('text/plain')
       .expect('Access-Control-Allow-Origin', '*')
+      .expect('Access-Control-Max-Age', '0')
```

Comment:
Ah, sorry. I just misinterpreted your suggestion to set the default `corsMaxAge` to `0` instead of `null` to mean we should send an `Access-Control-Max-Age` header with the value of `0` by default.

Actually, the default `corsMaxAge` should be `0` which will mean **no** header is sent, correct?